test-utils/src/monerod.rs

```diff
@@ -44,6 +50,7 @@ pub async fn monerod<T: AsRef<OsStr>>(flags: impl IntoIterator<Item = T>) -> Spa
         .arg(format!("--p2p-bind-port={}", p2p_port))
         .arg(format!("--rpc-bind-port={}", rpc_port))
         .arg(format!("--zmq-rpc-bind-port={}", zmq_port))
+        .arg(format!("--data-dir={}", db_location.display()))
```

Comment:
```suggestion
        .arg(format!(r"--data-dir='{}'", db_location.display()))
```
Not sure if necessary, but `tempfile` might return a PATH with spaces.

Reply:
Not using `tempfile` so I don't think this is necessary

Reply:
`temp_dir` is the same with `TMPDIR='/path/with space'` (I don't do that but someone might)

Reply:
I tested this with a space and it works without ``

Reply:
https://doc.rust-lang.org/std/process/struct.Command.html#method.arg
> This means that shell syntax like quotes, escaped characters, word splitting, glob patterns, variable substitution, etc. have no effect.

Yeah nvm I was wrong, too used to shell.